includes/course-theme/class-sensei-course-theme-templates.php

```diff
@@ -296,9 +296,26 @@ private function get_custom_templates() {
 			]
 		);
 
-		$db_templates = $db_templates_query->posts ?? [];
+		$db_templates         = $db_templates_query->posts ?? [];
+		$active_db_templates  = [];
+		$active_template_name = Sensei_Course_Theme_Template_Selection::get_active_template_name();
+		foreach ( $db_templates as $db_template ) {
+			$post_name = $db_template->post_name;
+
+			if ( strpos( $post_name, $active_template_name ) === false ) {
```

Comment:
This could be a bit fragile, matching templates that have the template name anywhere. Since we also needs to check the lesson/quiz below, something like `list( $post_type, $template_name) = explode('__', $post_name )` could work.

Reply:
Good suggestion. Updated 6c6b57641790df8dc7f2ce8eaea020171be93c37